packages/earn-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.5.0]
+
+### Added
+
+- Add pooled staking vault daily apys and vault apy averages to earn controller ([#5368](https://github.com/MetaMask/core/pull/5368))
```

Comment:
@Matt561 is this already in use in clients ? is it already in a live version ?

Reply:
I see there has been a change too where `vaultData` has been renamed tot `vaultMetadata`